<launch>
  <arg name="cmd_vel_in" default="in_twist"/>
  <arg name="cmd_vel_out" default="out_twist"/>

  <node pkg="obstacle_stop_planner" exec="obstacle_surround_stop_planner_node" name="obstacle_surround_stop_planner_node" output="screen">
    <param from="$(find-pkg-share obstacle_stop_planner)/config/obstacle_surround_stop_planner.param.yaml"/>
    <remap from="points_raw" to="velodyne_points"/>
    <remap from="cmd_vel_in" to="$(var cmd_vel_in)"/>
    <remap from="cmd_vel_out" to="$(var cmd_vel_out)"/>
  </node>
</launch>
